### DISCUSSION TITLE
Apple Watch platform

### DISCUSSION THREAD

**Discussion by Madmen777** [wojciech-kulik/xcodebuild.nvim#126] [Q&A] [answered]
is it possible to use xcodebuild for the apple watch platform? for example does build, test, and testing with simulator work correctly? thanks.

ps. is there some more info to fill all the fields of XcodebuildSetup? A detailed explanation would be good. For example if you enter the project I am not sure what is asked here. the project folder or file, and so on.

**Comment by wojciech-kulik** [comment, marked as answer]
Hi, `XcodebuildSetup` shows you just files to select, you don't have to enter it manually. If you don't see any items in the picker, then probably you didn't start Neovim from your project root folder.

Apple Watch is already supported on the main branch, please update the plugin if it doesn't work.

**Reply by wojciech-kulik** [reply]
@Madmen777 does it answer your question?

**Reply by Madmen777** [reply]
yes, you can close. thank you
